README.md

```diff
@@ -58,52 +39,30 @@ Unlike other SR v4 boards, the STM32 on the motor board does not directly commun
 
 ### Serial Commands
 
+Commands can be sent to the board via the serial or USB serial interface. Each command is its own line and is terminated with a new line.
 
-8 bit commands should be sent over the serial interface to the board.
-
-The following commands are supported:
-
-| Decimal Value | Unicode| Purpose                         |
-|---------------|--------|---------------------------------|
-| 0             | `\x00` | Reset the STM32                 |
-| 1             | `\x01` | Get the firmware version string |
-| 2             | `\x02` | Set Motor 0 value               |
-| 3             | `\x03` | Set Motor 1 value               |
-| 4             | `\x04` | Reset into USB bootloader       |
-
-
-### Setting Motor Speeds
-
-The value of a motor can be set by sending the appropriate command, and then a second byte as follows:
-
-The argument byte is a standard 8-bit signed number, where a few values are reserved for the special cases of no-op, coast and brake.
-
-| Decimal Value | Purpose         |
-|---------------|-----------------|
-| 0             | No-op           |
-| 1             | Coast the motor |
-| 2             | Brake the motor |
-
-### Firmware String
-
-When the firmware string is requested from the motor board, you will receive bytes of the following format over the serial interface: `MCV4B:3\n`.
-
-The number after the colon represents the firmware version of the board.
+Action | Description | Command | Parameter Description | Return | Return Parameters
+--- | --- | --- | --- | --- | ---
+Identify | Get the board type and version | *IDN? | - | Student Robotics:MBv4B:\<asset tag>:\<software version> | \<asset tag> <br>\<software version>
+Status | Get board status | *STATUS? | - | \<output faults>:\<input voltage> | \<output faults> - a comma separated list of 1/0s indicating if an output driver has reported a fault  e.g. 1,0<br>\<input voltage> - voltage at 12V input in mV
+Reset | Reset board to safe startup state<br>- Turn off all outputs<br>- Reset the lights | *RESET | - | ACK | -
+Set motor power | Sets the speed of one of the motors | MOT:\<n>:SET:\<value> | \<n> motor number, int, 0-1<br>\<value> motor power, int, -1000 to 1000 | ACK | -
+Read motor power | Gets the current speed setting of the motor | MOT:\<n>:GET? | \<n> motor number, int, 0-1 | \<enabled>:\<value> | \<enabled> motor enabled, int, 0-1 <br>\<value> motor power, int, -1000 to 1000"
+Disable motor output | Puts the motor into high impedance (equivalent to current coast) | MOT:\<n>:DISABLE | \<n> motor number, int, 0-1 | ACK | -
+Read motor current | Read the current power draw of the motor | MOT:\<n>:I? | \<n> motor number, int, 0-1 | \<current> | \<current> - current, int, measured in mA
+Enter bootloader | Enter the serial bootloader to load new firmware | *SYS:BOOTLOADER | - | ACK | -
```

Comment:
question: Does the `ACK` get sent before the bootloader starts?

Reply:
Yes